static/img/diagrams/case-management-system-context.iuml

```diff
@@ -25,10 +26,14 @@ Rel(data_insight_team, qlik, "Manage data extraction and reporting pipelines")
 Rel(qlik, social_care_cases, "Hourly data export")
 Rel(google, qlik, "Data ingestion via Google API")
 Rel(social_care_cases, google, "Authenticates with groups and view Google Docs")
+Rel(social_care_workflows, google, "Authenticates with groups and view Google Docs")
 Rel(external_referrers, google, "Submit referrals via public form")
 BiRel(google, mash_referral, "Proxy referrals. Generate google docs")
 Rel(mash_referral, social_care_cases, "Submit to")
 BiRel(social_care_finance, social_care_cases, "TBD")
+Rel(care_practitioners, social_care_workflows, "Start an assessment")
+BiRel(social_care_cases, social_care_workflows, "Share resident data")
+BiRel(social_care_finance, social_care_workflows, "TBD")
```

Comment:
is this the current production state?

Reply:
Yes, and the finance connection is still to be determined, but they are integrating.